tests/registerInvalidEmail.test.js

```diff
@@ -0,0 +1,27 @@
+const { test, expect } = require('@playwright/test');
+const { RegisterPage } = require('../pages/registerPage.js');
+import { generateRandomString } from '../utils/random.js';
+
+const randomUser = 'userForTest' + generateRandomString(4);
+const randomPassword = generateRandomString(8);
+const invalidEmail = 'userForTest' + generateRandomString(4) + '@com';
+
+test.describe('Redmine Registration', () => {
+    test('should register a new user', async ({ page }) => {
+        const registerPage = new RegisterPage(page);
+        await registerPage.goto('https://www.redmine.org/account/register');
+
+        const user = {
+            username: randomUser,
+            password: randomPassword,
+            confirmPassword: randomPassword,
+            firstname: 'TesTesterq',
+            lastname: 'Userenko',
+            email: invalidEmail,
+        };
+
+        await registerPage.registerUser(user);
```

Comment:
@karenn32  According to the TC1 ER - we must verify, that the fields have been filled, how is that covered here? There is no assertion to check the fields.